wine.py

```diff
@@ -0,0 +1,56 @@
+def score_wine_Collector_method(hand, standard_tokens_for_trail):
+  """Uses the Collector rule to calculate the number of points for wine tokens
+  
+  Each wine is worth 1 point for each different type of food you have at least one of
+
+  Could be called any time but to reflect the real game there is no running 
+  total of points; just a single calculation and comparison after the final ant 
+  has reached the anthill.
+
+  Parameters
+  ----------
+  standard_tokens_for_trail : (dict)
+    A dictionary containing the 'standard' tokens used to prepare the trail for this game. 
+
+    Used to cross-reference the player's hand to see which tokens will 
+    interact with the wine.
+    Keys are food types as strings
+    Values are integers.
+  
+  Returns
+  -------
+  wine_score : (integer)
+    An integer showing the player's total points from wine. 
+  """
+  wine_score = hand["wine"] *\
+    (len([food for food in hand\
+    if food in standard_tokens_for_trail and hand[food] > 0]))
+  return wine_score
+
+def score_wine_Oenophile_method(hand, standard_tokens_for_trail):
+  """Uses the Oenophile rule to calculate the number of points for wine tokens
+
+  Each wine is worth one point for each wineyou have.
+
+  Could be called any time but to reflect the real game there is no running 
+  total of points; just a single calculation and comparison after the final ant 
+  has reached the anthill.
+
+  Parameters
+  ----------
+  standard_tokens_for_trail : (dict)
+    A dictionary containing the 'standard' tokens used to prepare the trail for this game. 
+
+    Used to cross-reference the player's hand to see which tokens will 
+    interact with the wine. Not needed for this method but passed because the method 
+    which calles this also calls others where this is required. It is included here for concistency.
+    Keys are food types as strings
+    Values are integers.
+
+  Returns
+  -------
+  wine_score : (integer)
+    An integer showing the player's total points from wine. 
+  """
+  wine_score = hand["wine"] * hand["wine"]
```

Comment:
Optional change: Powers are done with `**`.
```suggestion
  wine_score = hand["wine"] ** 2
```

Reply:
I did check up on this first, and apparently n*n vs n**2 is not much different; but with smaller numbers n*n is slightly faster.